ezmodel/ezmodel.py

```diff
@@ -1,10 +1,52 @@
 def test_train_plot():
-    """"""
+    """
+    Creates plot of training and test error for an arbitrary sklearn model.
+
+    Args:
+        model (sklearn object): Previously initialized, untrained sklearn classifier or regression object
+                                    with fit & predict methods. Can also be a pipeline with several steps.
+
+        score_type (list or str): Should be one of: [mse, r2, adj_r2, auc, ...].
+                                      If a list, then a list containing several of those entries as elements.
+
+        x (ndarray): (n x d) array of features.
+        y (ndarray): (n x 1) Array of labels
+
+        hyperparameter (string): Hyperparameter of model to iterate over creating plot
```

Comment:
Type should be consistent with other arguments `(str)` rather than `(string)`, or `(string)` rather than `(str)` above